CHANGELOG.md

```diff
@@ -2,8 +2,8 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## Version [6.0.1][unreleased] - (in development)
-- TBA.
+## Version [6.1.0][unreleased] - (in development)
+- Changed: calling sync in preview is now deprecated, throwing an UnsupportedOperationException.
```

Comment:
I would say "disabled" here, because deprecated functionality is expected to still work, but its use is discouraged.